examples/schema/curation_rule-propagated.sssom.tsv

```diff
@@ -0,0 +1,16 @@
+#curie_map:
+#  HP: http://purl.obolibrary.org/obo/HP_
+#  MP: http://purl.obolibrary.org/obo/MP_
+#  orcid: https://orcid.org/
+#  DISEASE_MAPPING_COMMONS_RULES: https://w3id.org/sssom/commons/disease/curation-rules/
+#mapping_set_id: https://w3id.org/sssom/commons/examples/curation_rule.sssom.tsv
+#mapping_set_description: "This example illustrates how to express that all mapppings in a mapping set have been curated according to a specific curation rule"
+#license: "https://creativecommons.org/publicdomain/zero/1.0/"
+#creator_id: orcid:0000-0002-7356-1779
+#mapping_provider: "https://w3id.org/sssom/core_team"
+#curation_rule: DISEASE_MAPPING_COMMONS_RULES:MPR2
+#comment: This is an example file for the SSSOM for illustration only. Its contents are entirely fabricated.
+subject_id	predicate_id	object_id	mapping_justification

```

Comment:
The header for the fifth column is missing.

But in fact I don’t think the fifth column should even be there. The point of making `curation_rule` propagatable is to allow to specify the curation rule(s) once and for all, at the level of the mapping set, if the same rules are used by all mappings.

Even if that example was correct (if the column header was not missing), it would not demonstrate the usefulness of making `curation_rule` propagatable, since the three mappings do _not_, in this example, use the same rules.

So I’d suggest removing the fifth column entirely.